package.json

```diff
@@ -49,6 +49,10 @@
     "webpack": "^1.12.2",
     "webpack-dev-server": "^1.12.0"
   },
+  "peerDependencies": {
```

Comment:
It might be a bit unrelated to the main purpose of this PR, but I think it's good to specify which version of React the package requires in `peerDependencies` so consumers are properly warned if wrong version of React is in place. If it was added to `dependencies` instead you'd risk loading two versions of React and just having it in `devDependencies` would not give consumers any info about that.